tests/System.Text.Http.Tests/GivenIFormatterExtensionsForHttp.cs

```diff
@@ -65,7 +65,7 @@ public void The_http_extension_methods_can_be_composed_to_generate_the_http_mess
         //[Fact(Skip = "Issue https://github.com/dotnet/corefxlab/issues/599")]
         //public void WriteHttpHeader_resizes_the_buffer_taking_into_consideration_the_reserver()
         //{
-        //    _formatter = new BufferFormatter(32, FormattingData.InvariantUtf8, ArrayPool<byte>.Shared);
+        //    _formatter = new BufferFormatter(32, FormattingData.Utf8, ArrayPool<byte>.Shared);
```

Comment:
VS Refactor. Commented out test code will need to be fixed if someone re-enables it anyways.